portfolio/src/main/webapp/script.js

```diff
@@ -34,3 +34,34 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+var slideIndex = 1;
```

Comment:
- indexing beginning from 1 is unusual; most code I see is starting to count at 0